api/lb3app/server/datasources.local.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
   db: {
-    host: '127.0.0.1',
+    host: 'localhost',
```

Comment:
Hey @odin5on! I updated this info. I would rename your local schema to this to distinguish it from prod, and create this user in you database so you don't have to mess with this any more

Reply:
Was able to do this and got this exact branch running on my computer!